pkgs/development/python-modules/pymorphy3/default.nix

```diff
@@ -2,44 +2,54 @@
   lib,
   fetchFromGitHub,
   buildPythonPackage,
-  click,
-  dawg-python,
-  docopt,
-  pytestCheckHook,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  dawg2-python,
   pymorphy3-dicts-ru,
   pymorphy3-dicts-uk,
+
+  # tests
+  pytestCheckHook,
+  click,
 }:
 
 buildPythonPackage rec {
   pname = "pymorphy3";
-  version = "2.0.4";
-  format = "setuptools";
+  version = "2.0.6";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "no-plagiarism";
     repo = "pymorphy3";
     tag = version;
-    hash = "sha256-Ula2OQ80dcGeMlXauehXnlEkHLjjm4jZn39eFNltbEA=";
+    hash = "sha256-5s5v6+vMCeyn/73G5uUfRtTGkkqa7RNW8/r1v3Ay4us=";
   };
 
-  propagatedBuildInputs = [
-    dawg-python
-    docopt
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    dawg2-python
     pymorphy3-dicts-ru
     pymorphy3-dicts-uk
   ];
 
-  optional-dependencies.CLI = [ click ];
-
-  nativeCheckInputs = [ pytestCheckHook ] ++ optional-dependencies.CLI;
+  nativeCheckInputs = [
+    pytestCheckHook
+    click
+  ];
```

Comment:
```suggestion
    click
  ]
  ++ lib.flatten (lib.attrValues optional-dependencies);
```

Reply:
> ++ lib.flatten (lib.attrValues optional-dependencies);

Thanks that's a good idea. I wanted to simplify things but I don't mind it to be that way.